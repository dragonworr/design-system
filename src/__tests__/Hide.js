import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { Hide, theme } from '..'

describe('Hide', () => {
  test('renders', () => {
    const json = renderer.create(<Hide />).toJSON()
    expect(json).toMatchSnapshot()
  })
})
