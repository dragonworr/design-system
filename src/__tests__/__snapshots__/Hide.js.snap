// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hide renders 1`] = `
<div
  className=""
/>
`;

exports[`Hide renders with lg prop 1`] = `
@media screen and (min-width:48em) and (max-width:64em) {
  .c0 {
    display: none;
  }
}

<div
  className="c0 "
/>
`;

exports[`Hide renders with md prop 1`] = `
@media screen and (min-width:40em) and (max-width:48em) {
  .c0 {
    display: none;
  }
}

<div
  className="c0 "
/>
`;

exports[`Hide renders with sm prop 1`] = `
@media screen and (min-width:32em) and (max-width:40em) {
  .c0 {
    display: none;
  }
}

<div
  className="c0 "
/>
`;

exports[`Hide renders with xl prop 1`] = `
@media screen and (min-width:64em) {
  .c0 {
    display: none;
  }
}

<div
  className="c0 "
/>
`;

exports[`Hide renders with xs prop 1`] = `
@media screen and (max-width:32em) {
  .c0 {
    display: none;
  }
}

<div
  className="c0 "
/>
`;
